### DISCUSSION TITLE
Something went wrong. Please try again later.

### DISCUSSION THREAD

**Discussion by wangnanwy** [arc53/DocsGPT#1091] [General]
![QQ截图20240823161258](https://github.com/user-attachments/assets/e70607fe-c990-48cf-a97d-885c0cde289b)
I want to use the glm-4 model, but it doesn't work. The following prompt appears:
""
![QQ截图20240823160905](https://github.com/user-attachments/assets/25b76167-63af-4c2c-86a0-b725c99637c3)

**Comment by dartpain** [comment]
Hi I think I you case you have set api url to what looks like a openai compatable endpoint.
My suggestion would be:
Set `LLLM_NAME=openai`
Then inside `application/llm/openai.py`
```python
self.client = OpenAI(
            api_key=api_key,
            base_url="http://open.bigmodel.cn/api/paas/v4", 
        )
```
Alse in your env variables set MODEL_NAME to glm-4

API_URL should point to docsgpt backend.
Configuration of LLM endpoint is separate

Hope this helps

**Comment by wangnanwy** [comment]
已经收到您的来件！！
